util/counters.py

```diff
@@ -39,25 +47,29 @@ class CounterOptions(NamedTuple):
 
 
 class Counters():
-    '''Dictionary of named counters.
+    '''A dictionary of named counters for tracking metrics.
+
+    This class provides a flexible way to handle various types of counters,
+    including min/max values, debug counters, and periodic metrics like
+    processing rate, memory usage, and CPU time.
 
     Example usage:
       counters = Counters(prefix='my_process')
       ...
       counters.add_counter('input_rows')
-         .add_counter('output_rows', 10)
+      counters.add_counter('output_rows', 10)
```

Comment:
this was to illustrate chaining allowed

Reply:
reverted this change to the chaining calls.